### ISSUE TITLE
Cannot open data file for appended partition

### ISSUE THREAD

**Issue by Dragodraki** [PeyTy/xorriso-exe-for-windows#2]
Can you help me finding the bug here?
The xorriso command is from https://forums.ventoy.net/showthread.php?tid=1534

<img width="670" alt="Screenshot 2024-03-04 182031" src="https://github.com/PeyTy/xorriso-exe-for-windows/assets/76787321/18b94ede-19c6-45aa-a63c-d28fe5572e63">

**Comment by Dragodraki** [comment]
The same with another ISO like Testdisk...

**Comment by PeyTy** [comment]
Try to specify full file paths like `stdio:/cygdrive/C/your/path`

**Comment by Dragodraki** [comment]
Yes, I'll try as soon I have time for it. Currently there are some other things I've to do first.
I might be wrong here, but I suppose I already tried full path. Nevertheless I'll retry.
